src/daemon/daemon.cpp

```diff
@@ -432,6 +479,26 @@ grpc::Status mp::Daemon::launch(grpc::ServerContext* context, const LaunchReques
                                 grpc::ServerWriter<LaunchReply>* server) // clang-format off
 try // clang-format on
 {
+    if (metrics_opt_in == -2)
+    {
+        QFile opt_in_file{QDir(config->data_directory).filePath(metrics_opt_in_name)};
+        opt_in_file.open(QIODevice::WriteOnly);
+        opt_in_file.close();
+        metrics_opt_in = -1;
+        return grpc::Status(grpc::StatusCode::NOT_FOUND, "", "");
+    }
+    else if (metrics_opt_in == -1)
+    {
+        metrics_opt_in == QString::number(request->opt_in());
+        QFile opt_in_file{QDir(config->data_directory).filePath(metrics_opt_in_name)};
+        opt_in_file.open(QIODevice::WriteOnly);
+        opt_in_file.write(QString::number(request->opt_in()).toUtf8());
+        opt_in_file.close();
```

Comment:
We should also send a one-off "denied" metric here.

Reply:
How would a "denied" metric look like?

Reply:
Just a `{"denied": 1}` or some such? Just so that we can get a total number.

Reply:
Ok, seems [{"denied": 1}] is accepted by the endpoint, so we'll go with that:)

Reply:
As long as that will give us a sum of that ;)

Reply:
> As long as that will give us a sum of that ;)

Not really sure.  @cmars, will that `denied` like that give a total of all `denied`'s sent to the endpoint?